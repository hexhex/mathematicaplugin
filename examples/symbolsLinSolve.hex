matrix(1,1,"a"). matrix(1,2,"b").
matrix(2,1,"2a"). matrix(2,2,"d").

b(1,"e"). b(2,"f").

vektor(X,Y,Z) :- &linearSolve[matrix,b](X,Y,Z).
