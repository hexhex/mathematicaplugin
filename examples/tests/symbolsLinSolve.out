{vektor(1,1,"Times Power a -1 Power Plus Times 2 b Times -1 d -1 Plus Times -1 d e Times b f"), vektor(2,1,"Times Power Plus Times 2 b Times -1 d -1 Plus Times 2 e Times -1 f"), b(1,"e"), b(2,"f"), matrix(1,1,"a"), matrix(1,2,"b"), matrix(2,1,"2a"), matrix(2,2,"d")}
